src/commands/Tripulacion/tripulacion.js

```diff
@@ -0,0 +1,605 @@
+				'+tripulacion abandonar: Abandona la tripulación de la que formas parte [sólo para miembros]',
+				'+tripulacion establecer | <logo|historia|imagen> | <texto>: Solicita cambios en la ficha de tripulación [sólo para capitanes]',
+				'+tripulacion crear | <pirata> | <nombre de tripulación>: Crea una tripulación al mando del capitán pirata seleccionado [sólo para administradores]',
+				'+tripulacion moderar | [nombre de tripulación]: Revisa cambios en las fichas de tripulación [sólo para administradores]',
+				'+tripulacion borrar | <nombre de tripulación>: Borra a una tripulación y expulsa a sus miembros [sólo para administradores]',
+			].join("\n"),
+			comando: '+tripulacion <listar|detalles|reclutar|expulsar|abandonar|establecer|crear|moderar|borrar> | (argumentos)'
+		});
+
+		this
+			.createCustomResolver('pirata', (arg, possible, msg, [type]) => {
+				if (['listar', 'abandonar'].includes(type)) return undefined;
+
+				if (['detalles', 'borrar'].includes(type)) {
+					const customPossible = new Possible([undefined, "nombre de tripulación", "string", 1]);
+					if (!arg) arg = '';
+
+					return this.client.arguments.get('string').run(arg, customPossible, msg);
+				}
+
+				if (['moderar'].includes(type)) {
+					if (!arg) return undefined; // Without args, show a list of pending requests
+
+					const customPossible = new Possible([undefined, "nombre de tripulación", "string", 1]);
+					return this.client.arguments.get('string').run(arg, customPossible, msg);
+				}
+
+				if (['establecer'].includes(type)) {
+					if (!arg) arg = '';
+
+					arg = arg.toLowerCase();
+					if (!['logo', 'historia', 'imagen'].includes(arg))
+						throw "Debes elegir una de estas opciones: (logo, historia, imagen)";
+
+					return arg;
+				}
+
+				return this.client.arguments.get('member').run(arg, possible, msg);
+			})
+			.createCustomResolver('tripulacion', (arg, possible, msg, [type]) => {
+				if (['crear'].includes(type)) {
+					const customPossible = new Possible([undefined, "nombre de tripulación", "string", 1]);
+					if (!arg) arg = '';
+
+					return this.client.arguments.get('string').run(arg, customPossible, msg);
+				}
+
+				if (['establecer'].includes(type)) {
+					const customPossible = new Possible([undefined, "texto", "string", 1]);
+					if (!arg) arg = '';
+
+					return this.client.arguments.get('string').run(arg, customPossible, msg);
+				}
+
+				return undefined;
+			});
+	}
+
+	async listar(msg, []) {
+		// Gather general crew config vars
+		const crews = this.client.gateways.crews;
+		await crews.sync();
+
+		// Get sorted list of crews
+		const sortedList = crews.cache.sort();
+
+		// Early out if there aren't crews
+		if (sortedList.size <= 0)
+			throw "¡Yarr! Aún no hay ninguna tripulación mítica por estos mares...";
+
+		const embed = new MessageEmbed()
+			.setTitle('⚓ TRIPULACIONES MÍTICAS')
+			.setColor(0x2b9d98)
```

Comment:
Usa los números hexadecimales en mayúsculas:
```suggestion
			.setColor(0x2B9D98)
```